lms/resources/_lti_launch.py

```diff
@@ -32,7 +32,6 @@ def __init__(self, request):
         self._request = request
         self._authority = self._request.registry.settings["h_authority"]
         self._ai_getter = self._request.find_service(name="ai_getter")
-        self._hypothesis_config = None
```

Comment:
This was here because `LTILaunchResource.hypothesis_config` was mutable, and views could mutate it. It's now a sub-dict of `LTILaunchResource.js_config` which is also mutatable by views, so this is no longer needed